Makefile

```diff
@@ -33,6 +40,8 @@ docker-run-local-cloud:
 docker-run-cloud:
```

Comment:
Do we need `git` here, too?

Reply:
Yes @leecalcote , I have added that in my latest commit. Also, I think I have messed up my PR by doing a rebase with the master. Can someone help me out reverting this?